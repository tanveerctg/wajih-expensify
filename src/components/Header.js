import React from 'react';
import { NavLink } from 'react-router-dom';
const Header=()=>{
  return(
    <div>
      <h1>Expensify</h1>
      <NavLink to="/dashboard" activeClassName="selected" exact={true}>Dashboard</NavLink>
      <NavLink to="/create" activeClassName="selected">Create Expense</NavLink>
      <NavLink to="/help" activeClassName="selected">Help</NavLink>
    </div>
  )

}
export default Header;