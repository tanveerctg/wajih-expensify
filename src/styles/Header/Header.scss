.Header__1st{
  height:3rem;
  background-color: #2e0044;
  color:white;
  padding:1rem;
}
.Header__1st__inner{
  display:grid; 
  width:$desktop-view;
  margin-left: auto;
  margin-right: auto;
  grid-template-columns: 1fr auto;
}
.Header__btn{
  background: transparent;
  border:none;
  font-size: 1.2rem;
  color:whitesmoke;
  cursor: pointer;
}
.Header__2nd{
  height:8rem;
  background-color: #f3f5f6;
  color:white;
}
.Header__2nd__inner{
  display:grid; 
  width:$desktop-view;
  margin-left: auto;
  margin-right: auto;
  padding:1rem;
  grid-template-columns: 1fr;
  color:black;
}
.red{
  display:grid; 
  height:4rem;
  align-items:start;
}
.Header__2nd__btn{
  height:2rem;
  width:8rem;
  background-color:#2e0044;
  border:none;
  color:white;
  font-size: 14px;
  cursor:pointer;
}
.Header__2nd__btn:hover{
  border:1px solid #2e0044;
  background-color:#ebbfff;
  color:black;
}
.link__style__mainPage{
  color:white;
}


