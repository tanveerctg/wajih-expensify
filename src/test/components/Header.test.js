import toJson from 'enzyme-to-json';
import React from 'react';
import { shallow } from 'enzyme';
import Header from  '../../components/Header';
import setupTests from '../setupTests';


test("should render HEADER component correctly",()=>{
  const wrapper=shallow(<Header />);
  expect(toJson(wrapper)).toMatchSnapshot();
})