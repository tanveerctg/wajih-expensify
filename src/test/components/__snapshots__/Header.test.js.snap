// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HEADER component correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="selected"
    aria-current="page"
    exact={true}
    to="/"
  >
    Main page
  </NavLink>
  <NavLink
    activeClassName="selected"
    aria-current="page"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName="selected"
    aria-current="page"
    to="/help"
  >
    Help
  </NavLink>
</div>
`;
